readthedocs/oauth/models.py

```diff
@@ -168,8 +169,17 @@ class RemoteRepository(models.Model):
         null=True,
         blank=True,
     )
-
     json = models.TextField(_('Serialized API response'))
+    # TODO: Make remote_id and vcs_provider not nullable and
+    #  unique together after migration
+    remote_id = models.CharField(max_length=128, blank=True, null=True)
```

Comment:
I think we should index this field, since it's the one that we are going to use most. We indexed `full_name` for this reason.